htdocs/install/mysql/tables/llx_product_lot.sql

```diff
@@ -18,15 +18,20 @@
 -- ============================================================================
 
 CREATE TABLE llx_product_lot (
-  rowid           integer AUTO_INCREMENT PRIMARY KEY,
-  entity          integer DEFAULT 1,
-  fk_product      integer NOT NULL,				-- Id of product
-  batch           varchar(128) DEFAULT NULL,	-- Lot or serial number
-  eatby           date DEFAULT NULL,			-- Eatby date
-  sellby          date DEFAULT NULL, 			-- Sellby date
-  datec         datetime,
-  tms           timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-  fk_user_creat integer,
-  fk_user_modif integer,
-  import_key    integer
+  rowid               integer AUTO_INCREMENT PRIMARY KEY,
+  entity              integer DEFAULT 1,
+  fk_product          integer NOT NULL,           -- Id of product
+  batch               varchar(128) DEFAULT NULL,  -- Lot or serial number
+  eatby               date DEFAULT NULL,          -- Eatby date
+  sellby              date DEFAULT NULL,          -- Sellby date
+  datec               datetime,
+  tms                 timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  fk_user_creat       integer,
+  fk_user_modif       integer,
+  eol_date            datetime,                   -- End of life
+  manufacturing_date  datetime,
+  scrapping_date      datetime,                   -- Put to disposal
+  commissionning_date datetime,                   -- First use
+  qc_frequency        int,                        -- Quality control periodicity
```

Comment:
Are you sure this is on product_lot ? This looks better on product.

Reply:
The qc_frequency is normally on the product.
However, this can change over time depending on the products and regulations.
This is the reason why we added it on product and product_lot
@eldy

Reply:
This means there is a default value at a time t (this should be in product). 
When a new lot arrives, we can store the value so we keep information of what qc frequency was when lot appears the first time.
Then in future we may decide to change the value. This will change the default value (so value on product and new lot).
So qc_frequency on lot is a copy of default value from product. So before having this field in product_lot, we should have it first on product.

Here we have several fields for different features need. This make difficult to validate because some things are good and some not.
I recommand to make one PR for fields for different cases:
field on product_lot for quality control so qc_frequency and fk_qcstatus
another for traceability so  eol_date, manufacturing_date, scrapping_date

Reply:
> Here we have several fields for different features need. This make difficult to validate because some things are good and some not. 

I hear you and will try to be more cautious in the content of my future PRs (I’m still new in Dolibarr world).

> So before having this field in product_lot, we should have it first on product.

That’s why I put it in `product` *and* `product_lot` in the current PR

Reply:
@altairis-noe I agree with @eldy 
"qc_frequency on lot is a copy of default value from product. So before having this field in product_lot, we should have it first on product."

Reply:
And I think it is what I have done. Unless @eldy means I should have two PRs, one for each table ?